core/node/api_server/src/web3/namespaces/debug.rs

```diff
@@ -259,7 +259,11 @@ impl DebugNamespace {
         };
 
         let call_overrides = request.get_call_overrides()?;
-        let call = L2Tx::from_request(request.into(), MAX_ENCODED_TX_SIZE)?;
+        let call = L2Tx::from_request(
+            request.into(),
+            MAX_ENCODED_TX_SIZE,
+            false, // Even with EVM simulation enabled, calls must specify `to` field
```

Comment:
hmm, why?

Reply:
According to various Ethereum docs, like [Alchemy](https://docs.alchemy.com/reference/eth-call), and [the copied docs for `CallRequest`](https://github.com/matter-labs/zksync-era/blob/a5ac9c164bc0d16a879bef1cab1ccdaeac205bdc/core/lib/basic_types/src/web3/mod.rs#L783). Logically speaking, `to == None` corresponds to a deployment, and deployments don't work with the call execution mode in case of "pure" EraVM transactions (I've checked recently when [adding more unit tests for API server](https://github.com/matter-labs/zksync-era/pull/3001); e.g., load test transactions with non-zero deploys always revert in this mode). I'm not 100% sure, though; mechanically, it's easy to treat calls in the same way as transactions. @0xVolosnikov What's the correct choice here?

Reply:
Decided to keep `to` mandatory.